ptero_shell_command/implementation/celery_tasks/shell_command.py

```diff
@@ -87,7 +87,12 @@ def webhook(self, webhook_name, webhooks, **kwargs):
 
         if webhook_name in webhooks:
             task = self._get_http_task()
-            task.delay('PUT', webhooks[webhook_name], **kwargs)
+            urls = webhooks[webhook_name]
+            if not isinstance(urls, list):
```

Comment:
It may be appropriate to do this transformation in the `validators.py`.  There is already precedence of transforming the submitted data structure in `get_job_post_data()`.